samples/sdfg_api/cudnn_tasklet.py

```diff
@@ -46,15 +46,43 @@
     code='''
     // Set the current stream to match DaCe (for correct synchronization)
     cudnnSetStream(handle, __dace_current_stream);          
-    #define checkCUDNN(expression)                           \\
+
+    
+
+
+    
+    float alpha = 1.0, beta = 0.0;
+    cudnnConvolutionForward(handle, &alpha, xDesc, x,
+                            fDesc, f,
+                            convDesc, algo,
+                            workSpace, workSpaceSizeInBytes,
+                            &beta,
+                            yDesc, y);
+                            
+    ''',
+    # Global code (top of file, can be used for includes and global variables)
+    code_global='''
+    #include <cudnn.h>
+    cudnnHandle_t handle;
+    cudnnTensorDescriptor_t xDesc;
+    cudnnTensorDescriptor_t yDesc;
+    cudnnFilterDescriptor_t fDesc;
+    cudnnConvolutionDescriptor_t convDesc;
+    cudnnConvolutionFwdAlgo_t algo;
+    size_t workSpaceSizeInBytes = 0;
+    void* workSpace{nullptr};
```

Comment:
you could keep these inside `code` for the sample.

Reply:
Well it should go to init then. Or all of init also to code?

Reply:
all of init can go into the code for this sample, similarly to the CUBLAS sample